include/mcl_3dl/state_6dof.h

```diff
@@ -155,6 +155,34 @@ class State6DOF : public mcl_3dl::pf::ParticleBase<float>
   {
     rot_.normalize();
   }
+  size_t covDimension() const override
+  {
+    return 6;
+  }
+  float covElement(const State6DOF& e, const size_t& j, const size_t& k)
```

Comment:
```suggestion
  float covElement(const State6DOF& e, const size_t& j, const size_t& k) override
```

Reply:
I think we cannot use `override` for `covElement` as it is not `virtual` (it is a template method so it cannot be declared as virtual).